Assets/Prefabs/UI/Menus/LobbySelect.prefab

```diff
@@ -970,7 +731,7 @@ PrefabInstance:
       objectReference: {fileID: 0}
     - target: {fileID: 7072107538278606930, guid: e472bbf090cca566cae93eec176f8dff, type: 3}
       propertyPath: m_Name
-      value: Local
+      value: Private
```

Comment:
This isn't a private lobby, it's a local lobby currently. Could we leave in both private and local lobbies, please? Cuz if private means friends only then they'll be able to hop in.

Reply:
Now e have queue, friends only and local.